modules/navier_stokes/doc/content/source/fvbcs/WCNSFVSwitchableInletVelocityBC.md

```diff
@@ -0,0 +1,53 @@
+# WCNSFVSwitchableInletVelocityBC
+
+!syntax description /FVBCs/WCNSFVSwitchableInletVelocityBC
+
+The file is similar to [WCNSFVInletVelocityBC.md] but it allows the boundary conditions to be switched on/off.
+
+There are two options for specifying a component of the inlet velocity:
+
+- specifying a velocity postprocessor
+
+- specifying a mass flow rate postprocessor and a density functor. The functor is
+  usually a functor material property, defined by a [GeneralFunctorFluidProps.md].
+  The scaling factor can be used to account for projections if the inlet flow and
+  the surface are not aligned.
+
+
+This boundary condition works with postprocessors, which may be replaced by constant
+values in the input. The intended use case for this boundary condition is to be receiving its value from
+a coupled application, using a [Receiver.md] postprocessor.
```

Comment:
Now that you are directly inheriting from `WCNSFVInletVelocityBC`, let's remove this given it exists in the base class doc

Reply:
Thanks. Deleted